crates/oxc_syntax/src/module_record.rs

```diff
@@ -65,10 +65,21 @@ pub struct ModuleRecord {
     /// not including export * as namespace declarations.
     pub star_export_entries: Vec<ExportEntry>,
 
+    /// Local exported bindings
     pub exported_bindings: FxHashMap<CompactStr, Span>,
+
+    /// Local duplicated exported bindings, for diagnostics
     pub exported_bindings_duplicated: Vec<NameSpan>,
 
+    /// Reexported bindings from `export * from 'specifier'`
+    /// Keyed by resolved path
+    pub exported_bindings_from_star_export: DashMap<PathBuf, Vec<CompactStr>>,
```

Comment:
This is needed because we won't be able to distinguish local and reported bindings from this module.